github-runner-ami/packer/files/docker-buildx.sh

```diff
@@ -26,6 +26,6 @@ fi
 buildx_version="v0.7.1"
 buildx_binary="buildx-${buildx_version}.$(uname -s)-${architecture}"
 plugins_dir="/home/runner/.docker/cli-plugins"
-mkdir -pv "${plugins_dir}"
-curl -L "https://github.com/docker/buildx/releases/download/${buildx_version}/${buildx_binary}" -o "${plugins_dir}/${buildx_binary}"
-chown -R runner:runner "${plugins_dir}"
+sudo -u runner mkdir -pv "${plugins_dir}"
+sudo -u runner curl -L "https://github.com/docker/buildx/releases/download/${buildx_version}/${buildx_binary}" -o "${plugins_dir}/docker-buildx"
+sudo -u runner chmod a+x "${plugins_dir}/docker-buildx"
```

Comment:
`sudo` shouldn't be needed, as these scripts are already run via sudo https://github.com/apache/airflow-ci-infra/blob/31fa88c66307d07cf80f0f8a25c1422590ed67d1/github-runner-ami/packer/ubuntu2004.pkr.hcl#L148

Reply:
Oh read the code ash. Running as `runner`